docs/programming/kotlin/fundamentals/functions.md

```diff
@@ -0,0 +1,11 @@
+# Functions
+
+## Declaring a Function
+
+### Using Functions
+
+### Arguments & Parameters
+
+## Return Types
+
```

Comment:
Scope may be something you want to introduce too.

Reply:
See: https://cs61a.org/assets/slides/05-Environments.pdf. You don't need to go nuts into environment diagrams but I think the idea of frames is something you could introduce? It's hard and we'll most likely need to have multiple revisions for this.